### ISSUE TITLE
pug-as-jsx-loader is not good, but their readme looks better

### ISSUE THREAD

**Issue by skotchpine** [pugjs/babel-plugin-transform-react-pug#127]
https://github.com/bluewings/pug-as-jsx-loader

How can we make this repo look better?

**Comment by skotchpine** [comment]
Here's a live playground I put together (this doesn't include `pug-filters`, so we can't merge it): https://skotchpine.github.io/babel-plugin-transform-react-pug/

**Comment by ezhlobo** [comment]
@skotchpine I have to agree with you. I'd love to make this repository look better, but I struggle to do so. If you know anything specific, let's do that ;).

BTW, we also have REPL: https://react.pugjs.org. It was used to reproduce issues people met, but might work well as a documentation 🤔

**Comment by skotchpine** [comment]
yes definitely
